conans/client/command.py

```diff
@@ -72,10 +72,24 @@ def new(self, *args):
         parser.add_argument("-b", "--bare", action='store_true', default=False,
                             help='Create the minimum package recipe, without build() or package()'
                             'methods. Useful in combination with "package_files" command')
+        parser.add_argument("-cis", "--ci_shared", action='store_true', default=False,
```

Comment:
Shouldn't be defaulted to True? I don't want to write

-cis -cil -cio -ciw -ciu  in a new command